operators/multiclusterobservability/controllers/multiclusterobservability/multiclusterobservability_controller.go

```diff
@@ -350,20 +372,44 @@ func (r *MultiClusterObservabilityReconciler) SetupWithManager(mgr ctrl.Manager)
 
 	cmPred := predicate.Funcs{
 		CreateFunc: func(e event.CreateEvent) bool {
-			if e.Object.GetName() == config.AlertRuleCustomConfigMapName &&
-				e.Object.GetNamespace() == config.GetDefaultNamespace() {
-				config.SetCustomRuleConfigMap(true)
-				return true
+			if e.Object.GetNamespace() == config.GetDefaultNamespace() {
+				if e.Object.GetName() == config.AlertRuleCustomConfigMapName {
+					config.SetCustomRuleConfigMap(true)
+					return true
+				} else if _, ok := e.Object.GetLabels()[config.BackupLabelName]; ok {
+					// resource already has backup label
+					return false
```

Comment:
if user create a custom allowlist with backup label, the event will be filtered, which is not desired.

Reply:
Per our conversation, the placement controller will get an event and process them, so this will work as expected.